src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -604,27 +570,35 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
       _VERIFY(STATUS)
       call MAPL_GetPointer ( EXPORT, DryPLE1r8, 'DryPLE1r8',  RC=STATUS )
       _VERIFY(STATUS)
+      call MAPL_GetPointer ( EXPORT, SPHU0r8, 'SPHU0r8', RC=STATUS )
```

Comment:
Having _EXPORT as a suffix in addition to the suffix for imports would further help with clarity when reading the code.

Reply:
Added in #181 (merged into this PR)